cmake/external/onnxruntime_external_deps.cmake

```diff
@@ -212,9 +223,10 @@ set(protobuf_BUILD_TESTS OFF CACHE BOOL "Build protobuf tests" FORCE)
 set(protobuf_USE_EXTERNAL_GTEST ON CACHE BOOL "" FORCE)
 
 if (CMAKE_SYSTEM_NAME STREQUAL "Android")
-  set(protobuf_BUILD_PROTOC_BINARIES OFF CACHE BOOL "Build protobuf tests" FORCE)
+  set(protobuf_BUILD_PROTOC_BINARIES OFF CACHE BOOL "Build protoc" FORCE)
```

Comment:
does cross-compiling require an external protoc to be used? I would have assumed so as we can't use a protoc built for the target platform to generate files on the host.

if that is the case, should we maybe instead verify that `CMAKE_CROSSCOMPILING` being true implies that `ONNX_CUSTOM_PROTOC_EXECUTABLE` is set? then this `set(protobuf_BUILD_PROTOC_BINARIES OFF)` could be handled by the one above.